docs/UserGuide.md

```diff
@@ -158,6 +158,57 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+
+### Creating a new volunteer's profile : `vcreate` [coming soon]
+
+Volunteer Coordinators can create new volunteer profiles, and add the volunteer into the volunteer list.
+
+Format: `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL [s/SKILLS]…`
+
+Parameters:
+ * vn/ - Volunteer name
+ * hp/ - Phone number of the volunteer
+ * e/ - Email address of the volunteer
+ * s/ - Skills a volunteer may have
+
+<box type="tip" seamless>
+
+**Tip:** A volunteer can have any number of skills (including 0).
+</box>
+
+Restrictions:
+* The maximum number of characters of a volunteer name is 30.
+* The email must be in a valid format.
+* The phone number must be a valid 8-digit Singapore phone number.
+
+Examples:
+* `vcreate vn/John hp/91234567 e/john123@gmail.com` creates a volunteer named `John` with a phone number of `91234567` and an email address of `john123@gmail.com`, with no specific skills. The volunteer profile will be appended to the bottom of the volunteer list.
+* `vcreate vn/Mary hp/92345678 e/mary123@gmail.com s/Cooking s/Carrying heavy goods` creates a volunteer named `Mary` with a phone number of `92345678` and an email address of `mary123@gmail.com`, with two skills: `Cooking` and `Carrying heavy goods`. The volunteer profile will be appended to the bottom of the volunteer list.
+
+### Listing all volunteers : `vlist` [coming soon]
+
+Shows a list of all volunteers in the volunteer list.
+
+Format: `vlist`
+
+### Deleting a volunteer from the volunteer list: `vdelete` [coming soon]
+
+Volunteer coordinators can delete volunteers and remove them from the volunteer list if they no longer wish to volunteer anymore.
+
+Format: `vdelete VOLUNTEER_ID`
```

Comment:
Should we have `vdelete vid/VOLUNTEER_ID`  or stick to your current `vdelete VOLUNTEER_ID` ? Because for my edelete feature, I put the format as `edelete eid/EVENT_ID`

Reply:
hmm isn't the id itself just fine? like make the command as concise as possible i guess. but if you already did that i don't mind standardising

Reply:
I think we shd do urs without the `id` part ba. I'll change mine